VirtusLab-Open-Source/strapi-plugin-navigation
# feat: cache plugin integration

## Ticket

https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/issues/417

## Summary

What does this PR do/solve? 

- integration with [REST Cache Plugin](https://strapi-community.github.io/strapi-plugin-rest-cache/)

## Test Plan

- install cache plugin
- enable and provide configuration
- enable cache in plugin's config 
- read for a navigation via client call multiple time
- calls after the first one should be affected by caching (load times are lower)
- call cache should be the same for items specified in rest cache plugin's config and navigation calls
